internal/backend/auth/authsessions/config.go

```diff
@@ -3,26 +3,26 @@ package authsessions
 import (
 	"net/http"
 
-	"github.com/dghubble/sessions"
-
 	"go.autokitteh.dev/autokitteh/internal/backend/configset"
 )
 
 type Config struct {
-	Cookie     *sessions.CookieConfig
+	SameSite   http.SameSite
 	CookieKeys string `koanf:"cookie_keys"` // pairs of hash and block keys.
+	Domain     string `koanf:"ui_domain"`
+	Secure     bool
 }
 
 var Configs = configset.Set[Config]{
 	Default: &Config{
-		Cookie: func() *sessions.CookieConfig {
-			c := sessions.DefaultCookieConfig
-			c.SameSite = http.SameSiteNoneMode
-			return c
-		}(),
+		SameSite: http.SameSiteNoneMode,
+		Secure:   true,
+		Domain:   "autokitteh.cloud",
```

Comment:
i don't think this should appear is OSS. we should set it ourselves in terraform.

Reply:
It's just the default
It can be overridden by env variables